examples/features/effect-handlers.effekt.md

```diff
@@ -93,3 +97,65 @@ def genFibs(limit: Int): List[Int] / {} = {
 ```effekt:repl
 genFibs(15)
 ```
+
+## Singleton operation
+
+Often you want to declare an interface that is entirely defined by just one operation. In this case you can declare it as a singleton operation:
+
+```effekt:hide
+extern def time(): Int =
+  js "Date.now()"
+```
+
+```
+effect tell(): Int
+```
+
+The handler for this interface uses a slightly different more concise syntax like it is used in the literature:
+
+```
+def tellTime[A] { prog: () => A / tell }: A =
+  try { prog() }
+  with tell { () =>
+    resume(time())
+  }
+```
```

Comment:
This is slightly confusing as the `time` is invisible. Could we rename the extern def to something more primitive-looking? Something like
``````suggestion
```effekt:hide
extern def primitiveTimeNow(): Int =
  js "Date.now()"
```

```
effect tell(): Int
```

The handler for this interface uses a slightly different more concise syntax like it is used in the literature:

```
def tellTime[A] { prog: () => A / tell }: A =
  try { prog() }
  with tell { () =>
    resume(primitiveTimeNow())
  }
```
``````
(The name could probably still be improved...)

Also, this nitpick could be invalidated by us using the `time()` somewhere else before...

Reply:
As a different nitpick: it could be interesting to show why the `() =>` is needed (let the effect operation take an argument), but then that destroys this example :'(